data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,450 @@ mission "Unfettered returning home"
+			choice
+				`	(Go back to the shipyard.)`
+					goto leave
+				`	"Why would they mind a different ship?"`
+			`	"As this ship is ancient they will try to preserve it."`
+			label leave
+			`	They walk away until that you cannot hear them speak anymore, leaving you with the access codes to the ship so you can bring it to the shipyard. With the aid of the autopilot, you manage to bring it without any complications.`
+				accept
+			label foolish
+			`	You get your weapon out and proceed to shoot on the ship.`
+			branch miss
+				random < 50
+			branch lucky?
+				random < 2
+			`	Your shot reaches the ship, but the small weapon has no chance of doing anything to the heavy armor, even if it is somewhat worn. The ship then shoots back at you, barely missing and causing a geyser of sand to erupt at you and force you to drop your weapon. One of the two Hai then approaches you, seemingly disappointed.`
+			`	"You're lucky to still be alive, but even so you are of no use to us if you think it wise to use your firearm on a spaceship. Farewell." They turn and leave without another word, and the ship flies off.`
+				decline
+			label lucky?
+			`	You manage to land a shot, although it seems to do nothing to the large spacecraft before you. Then you hear a small explosion. You aren't sure whether the damaged ship's systems failed on their own, or if by some stroke of luck, your shot did something to it.`
+			`	Unfortunately, you do not get a chance to ponder it, as the ship abruptly explodes, taking you and everyone else on the beach down with it.`
+				die
+			label miss
+			`	You miss your shot. Neither the ship nor the two Hai react. You feel like you heard a chuckle, but you're not sure where it came from.`
+				goto "try again"
+	on defer
+		set "Hai Legacy 1: deferred"
+	on accept
+		clear "unfettered: ocean dialog"
+		clear "unfettered: pilot dialog"
+		log "Accepted a somewhat shady deal for an old Hai ship, an Anomalocaris, that seems to still be in good functioning state."
+
+
+
+mission "Hai Legacy 2"
+	autosave
+	landing
+	name "Heritage Reclaimed"
+	description "Use the newly acquired Anomalocaris to bring aid to the Unfettered Hai on <planet> by <date>."
+	clearance
+	deadline 7
+	cargo "food (aid)" 121
+	source "Hai-home"
+	destination "Firelode"
+	to offer
+		not "Hai Legacy 1: declined"
+		has "First Contact: Unfettered: offered"
+		"reputation: Hai (Unfettered)" < 0
+		not "Unfettered Jump Drive 1: offered"
+		"credits" >= 10000000
+	to accept
+		has "ship model: Anomalocaris"
+	on offer
+		conversation
+			`Upon returning to the spaceport, a worker immediately greets you while others begin to fill your ship with crates. A quick check of the shipment data shows that the cargo has already been allocated in your name. All that's left is to deliver it to <destination> by <date> for a promised sum of <payment>.`
```

Comment:
```suggestion
			`You leave the shipyard and go to the spaceport after your encounter. A worker immediately greets you while others begin to fill your ship with crates, and a quick check of the shipment data shows that the cargo has already been allocated in your name. All that's left is to deliver it to <destination> by <date> for a promised sum of <payment>.`
```
We weren't necessarily at the spaceport before. We went to the shipyard. It could be that the spaceport isn't connected to the shipyard, for example. So it feels odd when this automatically pops up when I leave the shipyard, but didn't click the spaceport button. Adjusted the sentence a bit for the change.